cli/src/commands/diffedit.rs

```diff
@@ -64,6 +64,9 @@ pub(crate) struct DiffeditArgs {
     /// Specify diff editor to be used
     #[arg(long, value_name = "NAME")]
     tool: Option<String>,
+    /// Do not modify the content of the revision specified with `-r` or `--to`
```

Comment:
Let's try to clarify this. I'm not sure most readers will understand what it means. (It also seems incorrect in that it says that the target commit itself is unchanged). Maybe something like this:

> Preserve the content (not diff) when rebasing descendants
> 
> When rebasing a descendant on top of the rewritten revision, its diff compared to its parent(s) is normally
> preserved, i.e. the same way that descendants are always rebased. This flag makes it so the content/state
> is preserved instead of preserving the diff.

Do you think that's clear?

Reply:
The "(not diff)" looked a bit strange to me at first glance, but with the full explanation it makes sense.

Reply:
not sure if it actually helps, but "(not the diff)" sounds better upon rereading it

Reply:
Yes a bit indeed. Updated.